packages/ses/test/test-compartment-instance.js

```diff
@@ -28,22 +28,7 @@ test('Compartment instance', t => {
   );
 
   t.is(c.toString(), '[object Compartment]', 'toString()');
-  t.is(c[Symbol.toStringTag], undefined, '"Symbol.toStringTag" property');
+  t.is(c[Symbol.toStringTag], 'Compartment', '"Symbol.toStringTag" property');
 
   t.deepEqual(Reflect.ownKeys(c), [], 'static properties');
-  t.deepEqual(
-    Reflect.ownKeys(Object.getPrototypeOf(c)).sort(),
-    [
-      'constructor',
-      'evaluate',
-      'globalThis',
-      'import',
-      'importNow',
-      'load',
-      'module',
-      'name',
-      'toString',
-    ].sort(),
-    'prototype properties',
-  );
```

Comment:
This appears to be redundant with the prototype properties test performed in `test-compartment-prototype`. Checking that the prototype of the Compartment instance is the expect prototype ought to be enough for this test?

Reply:
I buy it.